dalle/models/__init__.py

```diff
@@ -1,6 +1,6 @@
 # ------------------------------------------------------------------------------------
 # Minimal DALL-E
```

Comment:
I just found that this (Minimal DALL-E) should be fixed to minDALL-E. Could you update this accordingly? Many thanks!